open-sphere-base/overlay/src/main/java/io/opensphere/overlay/util/MousePositionUtils.java

```diff
@@ -0,0 +1,49 @@
+package io.opensphere.overlay.util;
+
+import java.awt.Point;
+
+import io.opensphere.core.Toolbox;
+import io.opensphere.core.math.Vector2i;
+import io.opensphere.core.model.Altitude;
+import io.opensphere.core.model.GeographicPosition;
+
+/**
+ * Utility class relating to the mouse position on the screen.
+ */
+public class MousePositionUtils
+{
+    /** The current position of the mouse on the globe. */
+    private static GeographicPosition currentPosition;
```

Comment:
Since currentPosition is set and get from multiple threads, it should be `volatile`.